WordPress/src/main/java/org/wordpress/android/ui/reader/discover/viewholders/ReaderRecommendedBlogViewHolder.kt

```diff
@@ -46,4 +76,13 @@ class ReaderRecommendedBlogViewHolder(
             imageManager.cancelRequestAndClearImageView(siteIcon)
         }
     }
+
+    data class ReaderRecommendedBlogBinding(
+        val root: View,
+        val siteName: TextView,
+        val siteUrl: TextView,
+        val siteDescription: TextView,
+        val siteIcon: ImageView,
+        val siteFollowIcon: ReaderFollowButton,
```

Comment:
💡 Suggestion: change this to `siteFollowButton` since the new style doesn't have an icon anymore.

Reply:
Good catch! Renamed.